### DISCUSSION TITLE
Removing black background

### DISCUSSION THREAD

**Discussion by smtabatabaie** [hvianna/audioMotion.js#65] [Q&A]
Hi, Thanks for this great tool. Is it possible to move the black background of the visualizer?

**Comment by hvianna** [comment]
You can set the [Background](https://audiomotion.app/docs/#/users-manual?id=background) to display the album cover or a background image, or create a [custom gradient](https://audiomotion.app/docs/#/users-manual?id=gradient-editor) with a different background color.

Moving this to Discussions.
